cg/cli/set/base.py

```diff
@@ -163,41 +163,27 @@ def list_changeable_sample_attributes(
 
 def show_set_sample_help(sample_obj: models.Sample = "None") -> None:
     """Show help for the set sample command"""
-    LOG.info("sample_id: optional, internal_id of sample to set value on")
-    show_option_help(long_name=OPTION_LONG_SKIP_LIMS, help_text=HELP_SKIP_LIMS)
-    show_option_help(short_name=OPTION_SHORT_YES, long_name=OPTION_LONG_YES, help_text=HELP_YES)
-    show_option_help(
-        short_name=OPTION_SHORT_KEY_VALUE, long_name=OPTION_LONG_KEY_VALUE, help_text=HELP_KEY_VALUE
-    )
-    list_changeable_sample_attributes(sample_obj, skip_attributes=NOT_CHANGEABLE_SAMPLE_ATTRIBUTES)
-    LOG.info(f"To set apptag use '{OPTION_SHORT_KEY_VALUE} application_version [APPTAG]")
-    LOG.info(f"To set customer use '{OPTION_SHORT_KEY_VALUE} customer [CUSTOMER]")
-    LOG.info(
-        f"To set priority use '{OPTION_SHORT_KEY_VALUE} priority [priority as text or " f"number]"
-    )
-
-
-def show_option_help(short_name: str = "", long_name: str = "", help_text: str = "") -> None:
-    """Show help for one option"""
-    help_message = f"Use "
-
-    if short_name:
-        help_message += f"'{short_name}'"
-
-    if short_name and long_name:
-        help_message += " or "
+    LOG.info(f"Below is a set of changeable sample attributes, to combine with -kv flag:\n")
 
-    if long_name:
-        help_message += f"'{long_name}'"
+    list_changeable_sample_attributes(sample_obj, skip_attributes=NOT_CHANGEABLE_SAMPLE_ATTRIBUTES)
```

Comment:
```suggestion
    list_changeable_sample_attributes(sample_obj=sample_obj, skip_attributes=NOT_CHANGEABLE_SAMPLE_ATTRIBUTES)
```
Use named arguments if there are more than 2 arguments

Reply:
Done!